### DISCUSSION TITLE
Want to customize Column at View Assets for users

### DISCUSSION THREAD

**Discussion by rishibity** [snipe/snipe-it#18000] [Q&A]
### Snipe-IT Version

v5.4.1 build 6746 (g3e22dce11)

### Operating System

Windows

### Web Server

IIS

### PHP Version

7.4.3

### Is your feature request related to a problem? Please describe. A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

Hey 

I'm having a situation where I want to show users a customized view of their assigned assets. But I don't know where to do that. Please help me with this. I'm getting attached Columns options only.

![image](https://user-images.githubusercontent.com/99635338/165451470-c5a63c47-484e-471f-ad6c-97b4e503bdeb.png)
 

### Describe the solution you'd like A clear and concise description of what you want to happen.

![image](https://user-images.githubusercontent.com/99635338/165450610-51554f40-f2df-45ce-b226-cb96ae139d8c.png)

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.

**Comment by ReachyBayern** [comment]
Maybe you can also provide a solution to sort the columns also by users to have a custom view per user or as default view

**Reply by snipe** [reply]
`resources/views/account/view-assets.blade.php` is the blade for the user's view of their own assets
